### PR TITLE
fix(chat): Publish tooltips (Issue #1447, Issue #1463)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Tooltip for path inside "Publish to" text bot is not displayed
currently tooltip pointing to the center of the textbox
Note:Different font sizes for tooltips for Publish to and prompt/conversation name on the publication request details

Issues:

- Issue #1447 
- Issue #1463 

**UI changes**

Снимок экрана 2024-06-01 в 23 43 45
Снимок экрана 2024-06-01 в 23 43 54
Снимок экрана 2024-06-01 в 23 44 39
Снимок экрана 2024-06-01 в 23 44 54

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx [hunk 1]
```diff
@@ -192,7 +192,8 @@ const PromptView = ({ item: prompt }: PromptViewProps) => {
       <IconBulb size={18} className="text-secondary" />
       <Tooltip
         tooltip={prompt.name}
-        triggerClassName="truncate text-center w-full"
+        contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+        triggerClassName="block max-h-5 truncate whitespace-pre"
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx]: `block max-h-5` do we really need it here?

Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx]: thanks for the comment, I fixed the styles

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 2]
```diff
@@ -360,13 +360,21 @@ export function PublishModal({
...
                 >
-                  <span className="truncate">
-                    {constructPath(t(PUBLISHING_FOLDER_NAME), path)}
-                  </span>
-                  <span className="text-accent-primary">{t('Change')}</span>
+                  <div className="flex w-full justify-between truncate whitespace-pre break-all">
+                    <Tooltip
+                      tooltip={constructPath(t(PUBLISHING_FOLDER_NAME), path)}
+                      contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+                      triggerClassName="block max-h-5 truncate whitespace-pre"
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: do we need `triggerClassName` property?

Comment (Derikyan) [reply apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I fixed it, thanks for the comment

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/481963)

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 3]
```diff
@@ -374,13 +374,21 @@ export function PublishModal({
                   {t('Publish to')}
                 </label>
                 <button
-                  className="input-form mx-0 flex grow items-center justify-between rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
+                  className="input-form mx-0 flex grow items-center rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: most of this classes already in `button button-secondary`

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 4]
```diff
@@ -374,13 +374,21 @@ export function PublishModal({
...
-                    {constructPath(t(PUBLISHING_FOLDER_NAME), path)}
-                  </span>
-                  <span className="text-accent-primary">{t('Change')}</span>
+                  <div className="flex w-full justify-between truncate whitespace-pre break-all">
+                    <Tooltip
+                      tooltip={constructPath(t(PUBLISHING_FOLDER_NAME), path)}
+                      contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+                      triggerClassName="truncate whitespace-pre"
+                    >
+                      <span className="truncate">
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: why do we have `truncate` twice for trigger?

File: apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx [hunk 5]
```diff
@@ -193,8 +193,8 @@ const PromptView = ({ item: prompt }: PromptViewProps) => {
       <IconBulb size={18} className="text-secondary" />
       <Tooltip
         tooltip={prompt.name}
-        contentClassName="max-w-[400px]"
-        triggerClassName="truncate text-center w-full"
+        contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+        triggerClassName="truncate whitespace-pre"
       >
         <div className="truncate whitespace-pre break-all text-left">
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx]: why do we have `truncate` twice for trigger?

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx (hunk 3 above)
Comment (Derikyan) [reply apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I used input-form and button to reduce the repeating styles

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx (hunk 4 above)
Comment (Derikyan) [reply apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I removed the span

File: apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx (hunk 5 above)
Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx]: I removed the div

**Review by IlyaBondar — CHANGES_REQUESTED** [review]
image

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1476.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**